docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -161,6 +161,16 @@ values. Typical usage does not require you to configure them.
   - Maximum number of metastore data objects per transaction in the Hive
     metastore cache.
   - `1000`
+* - `delta.metastore.store-table-metadata-enabled`
+  - Store table comments and colum definitions to the metastore. The write
+    permission is required to update the metastore.
```

Comment:
what happens if we dont have the write permission?

Reply:
It will result in no-op with warning log.

Reply:
This means that we can flood the logs with these warnings for a correctly configured system that previously was working just fine. I'd recommend to provide some permission detection mechanism. If we can read from metastore fine, but we can't update the state there (after some reasonable number of samples to cater for transient errors like network), we should auto-disable the feature. This will avoid flooding logs, and also avoid spending cycles and api calls on code path that doesn't work anyway. On top of that we can have time-based expiration so that the system eventually auto-reconfigures if permissions change externally.  The duration of such circuit breaker would depend on how costly it is to detect lack of permissions.